src/widget/wtrackmenu.cpp

```diff
@@ -1725,10 +1733,19 @@ void WTrackMenu::slotRemoveFromDisk() {
         delListWidget->addItems(locations);
         mixxx::widgethelper::growListWidget(*delListWidget, *this);
         // Warning text
```

Comment:
This comment is pointless now.

Reply:
This and the comments below are meant to structure the dialog setup.
I removed this on because the string name is directly below.